CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased](https://github.com/contentful/contentful-laravel/compare/1.0.0...HEAD)
 
```

Comment:
Can you add the comment "this release contains breaking changes [...]" here, thank